public/application.js

```diff
@@ -58,6 +67,9 @@ document.addEventListener('DOMContentLoaded', (event) => {
 
         setTimeout(() => this.removeConfetti(), 12000);
       },
+      searchSparkle: function(name){
+console.log(name)
+      },
```

Comment:
I think I can see the direction that you were going with this change. I'm glad that you found the [v-model](https://v2.vuejs.org/v2/guide/forms.html#Text) to figure out how to bind the changes in the HTML to the JavaScript code. Do you think that we still need this function?